examples/basic_mysql_connection/index2.js

```diff
@@ -0,0 +1,30 @@
+var fs = require('fs');
+var mysql = require('mysql');
+
+var config = JSON.parse(fs.readFileSync("config-secret.json"))
+
+var connection = mysql.createConnection({
+		host: config.host,
+		user: config.user,
+		password: config.password,
+		port: config.port,
+		database: config.database
+});
+
+
+const sqlStatments=[
+				'INSERT INTO todos (Name, Done) VALUES ("Odai",1)',

```

Comment:
The homework asked you to `Write the necessary SQL statements to complete the functionality of your application`. These queries use hardcoded values, so they're probably not very useful for an application (yet).

However, you have achieved *some* of the functionality an application like this would need, like setting the status to "in progress". But you have no query for marking the todo as completed!